setup.py

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env python
 
-# SPDX-FileCopyrightText: 2017 Fermi Research Alliance, LLC
-# SPDX-FileCopyrightText: 2017 Fermi Research Alliance, LLC
 # SPDX-FileCopyrightText: 2019 Fermi Research Alliance, LLC
```

Comment:
These (4, 5) are the actual lines to remove. The date to keep is 2017

Reply:
This look better?

Reply:
perfect!